nemo_rl/distributed/worker_groups.py

```diff
@@ -544,6 +591,49 @@ def group_count(self) -> int:
         """Number of tied worker groups."""
         return len(self.tied_workers_groups)
 
+    def run_single_group_single_data(
+        self,
+        method_name: str,
+        group_idx: int,
+        *args,
+        run_rank_0_only_axes: list[str] | None = None,
+        **kwargs,
+    ) -> list[ray.ObjectRef]:
+        """Run a method on a single tied worker group with the same data.
+
+        Args:
+            method_name: Name of the method to call on each worker
+            group_idx: Index of the tied worker group to run on
```

Comment:
is it possible to deprecate all notions of 'tied worker group' and just refer to these workers by shard indices?

Reply:
@terrykong fixed in https://github.com/NVIDIA-NeMo/RL/pull/506/commits/4ae53514523978e0c8321343dc0a8a603c4693ea